Editor/Resources/EditorWindow/GameLiftPlugin.uss

```diff
@@ -0,0 +1,59 @@
+/* Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved. */
+/* SPDX-License-Identifier: Apache-2.0 */
+
+:root {
+    height: 100%;
+    width: 100%;
+    
+    --color-menu-background: #333333;
+    --color-content-background: #3F3F3F;
+    --color-button-primary: #2C5D87;
+    --color-button-hover: #386082;
+    
+    --font-size-menu: 18px;
+}
+
+.main {
+    flex-direction: row;
+    height: 100%;
+    width: 100%;
+}
+
+.main__menu {
+    width: 240px;
+    padding-top: 20px;
+    background-color: var(--color-menu-background);
+}
+
+.main__content {
+    flex-grow: 1;
+    background-color: var(--color-content-background);
+}
+
+.TabButton {
```

Comment:
we should be consistent with css class naming. On this file i see the following conventions
* TabButton (capitalization)
* main (capitalization)
* main__content (__)
* TabButton--selected (--)


Please pick one notation and ensure that all uss files stick to this.

Reply:
I understand `--selected` is a recognized way of adding identifiers in CSS so that seems fine

Reply:
Happy to follow BEM, I missed these and will refactor now